src/codegen.cpp

```diff
@@ -5078,7 +5085,7 @@ static std::unique_ptr<Module> emit_function(jl_method_instance_t *lam, jl_code_
 
 // --- initialization ---
 
-static MDNode *tbaa_make_child( const char *name, MDNode *parent, bool isConstant=false )
+MDNode *tbaa_make_child(const char *name, MDNode *parent, bool isConstant=false)
 {
     MDNode *n = mbuilder->createTBAANode(name,parent,isConstant);
```

Comment:
`MDBuilder mbuilder(parent->getContext())`

the `MDBuilder` class is just a single pointer (to a LLVMContext), it doesn't need to be a global